### ISSUE TITLE
assignment 4 Relax NG

### ISSUE THREAD

**Issue by rmz14** [ebeshero/DHClass-Hub#14]
Started the next assignment but know there are some errors. Please let me know if there are any suggestions. There is an rnc and xml document named as "Zabelsky" in the troubleshooting folder. I think I posted this stuff correctly now. 

Thanks, Ryan

**Comment by ebeshero** [comment]
@rmz14 I think you hit "revert" instead of "commit"--because instead of pushing new files into Troubleshooting, you undid the last thing I did (which was to remove a file extension from the readme.) I just reverted your revert! But try it again, make sure you hit commit and sync (and skip the revert button). I think I answered on your earlier questions here: https://github.com/ebeshero/DHClass-Hub/issues/13 

Does that help?
